google-login-plugin/src/com/google/gct/login/stats/UsageTrackerService.java

```diff
@@ -15,52 +15,28 @@
  */
 package com.google.gct.login.stats;
 
-import com.intellij.openapi.util.KeyedExtensionCollector;
-import com.intellij.util.PlatformUtils;
+import com.intellij.openapi.components.ServiceManager;
 
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 /**
- * Usage Tracker Service for obtaining extensions declared in plugin.xml with {@link
- * UsageTrackerExtensionPointBean}
+ * Usage Tracker Service for obtaining UsageTracking implementations
  */
-public final class UsageTrackerService {
+public abstract class UsageTrackerService {
```

Comment:
This is really a usage tracker provider. I'd expect a 'service' to actually be doing the thing.